modules/mono/editor_templates/EditorScript/basic_editor_script.cs

```diff
@@ -6,9 +6,10 @@
 [Tool]
 public partial class _CLASS_ : _BASE_
 {
+    // Called when the script is executed (using File -> Run in Script Editor).
     public override void _Run()
     {
-        // Called when the script is executed (using File -> Run in Script Editor).
+
```

Comment:
It would likely be better to just leave the block empty (0 empty lines) instead of having 1 empty line. When using a C# IDE you can click on the right of the opening brace and press enter to get the correct indentation.